docs/pages/more/expo-cli.mdx

```diff
@@ -315,6 +315,20 @@ The command `npx expo install expo-camera` and `npx expo install expo-camera --f
 
 <Terminal cmd={['$ npx expo install --fix']} />
 
+### Configuration
+There may be circumstances where you need to use a version of a package that is outside the range of safe versions recognized by `npx expo install`. In SDK 49 and higher projects, you can exclude specific packages from the version checks, while still allowing the `install` command to install, check, and fix any other dependencies. This configuration extends to the checking done by `npx expo-doctor`, suppressing warnings related to these specific packages during the Expo Doctor step in EAS Build.
```

Comment:
```suggestion
There may be circumstances where you want to use a version of a package is different from the version recommended by `npx expo install`. For example, you might test a new version of `react-native-reanimated`, verify that it works well in your app and fixes a bug that you encountered, and now you want to 1) not be warned by `npx expo start` or `npx expo-doctor` and 2) not have that package version changed when you run `npx expo install --fix`. 

In SDK 49 and higher projects, you can exclude specific packages from the version checks, while still allowing the `install` command to install, check, and fix any other dependencies. This configuration extends to the checking done by `npx expo-doctor`.
```

Reply:
My suggestion on phrasing the example part with a line break (not having line break, + longer paragraphs, are hard to skim and understand at the same time:

```markdown
For example, you want to test a new version of `react-native-reanimated` to verify that it works well in your app and fixes a bug that you encountered. Now you want 1) not be warned by `npx expo start` or `npx expo-doctor` and 2) not have that package version changed when you run `npx expo install --fix`.
```

I'd also suggest to remove the `,` after checks (not grammatically correct) and no need for pause here:

```markdown
...specific packages from the version checks while still...
```